.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+# Python CircleCI 2.0 configuration file
```

Comment:
this file still needs some more work to be done.
Tests are not running on Circle CI.
Key components don't seem to be running too.
You can look at [this](https://github.com/andela/dashiki-healthchecks/blob/develop/circle.yml) file for reference;